### PR TITLE
Fix PR screen in github

### PR DESCRIPTION
**Pull request by jgkamat**
Like before, please regenerate the CSS for me, I don't think I have node set up correctly.

Before:

2017-10-08-012049_1121x703_scrot

After:

2017-10-08-012037_1118x704_scrot

### CODE REVIEW

**Comment by jgkamat** [comment]
I'm overloading this to fix some PR link colors as well:

Before:
2017-10-08-012524_720x233_scrot

After:
2017-10-08-012643_695x176_scrot

**Comment by The-Compiler** [comment]
Hey @jgkamat! 😆 

This fixes at least some of the issues I mentioned in #5 🎉

**Comment by jgkamat** [comment]
Might as well keep going then! :D 
Before:
2017-10-08-153330_847x564_scrot
After:
2017-10-08-153254_826x539_scrot

Before:
2017-10-08-161635_1038x918_scrot
After:
2017-10-08-161614_1066x912_scrot

File: sites/github.styl [hunk 1]
```diff
@@ -12,8 +12,8 @@
 
 // ** Misc
 
+.link-gray-dark
 a.link-gray-dark
```
Comment (alphapapa) [inline sites/github.styl]: Since you added the plain class selector, can we remove the `a.link-gray-dark` selector?

File: sites/github.styl [hunk 2]
```diff
@@ -251,6 +251,9 @@ a.filter-item
     > h3
         background-color color-background-highlight-extra-less
         color emphasized
+.more-repos
+    background-color color-background-highlight-extra-less
+    box-shadow inset 0 1px 0 color-background-highlight-extra
```
Comment (alphapapa) [inline sites/github.styl]: Do you need to add `i` at the end to make it `!important`?

File: sites/github.styl [hunk 3]
```diff
@@ -269,14 +272,37 @@ a.filter-item
 .branch-action-state-dirty
 .branch-action-state-unstable
 .branch-action-state-unknown
-    .branch-action-icon
+.branch-action-icon
```
Comment (alphapapa) [inline sites/github.styl]: Why did you remove the indentation here?  IIUC it will now make all `.branch-action-icon` icons yellow.

File: sites/github.styl [hunk 4]
```diff
@@ -269,14 +272,37 @@ a.filter-item
...
         background-color yellow i
         color white i
 
+.text-shadow-light
+.branch-group-name
+.branch-summary
+    color color-text i
```
Comment (alphapapa) [inline sites/github.styl]: The `color` mixin applies `color-text i` automatically, so you can change this to just `color`.

File: sites/github.styl [hunk 5]
```diff
@@ -269,14 +272,37 @@ a.filter-item
...
+.branch-group-heading
 .branch-infobar
+.branch-name
     border-color comment
     background-color highlight
 
+.State
+.State:visited
+.octoicon-git-pull-request
+    color color-background-highlight i
```
Comment (alphapapa) [inline sites/github.styl]: The `color` mixin applies `i` automatically, so it can be left off here.

File: sites/github.styl [hunk 6]
```diff
@@ -583,10 +609,13 @@ a.label
         color transparentify(green, color-background, 0.65)
 
 .octicon-alert
-.octicon-check
     color white
 .octicon-x
     color red
+.octicon-check
+    color green
```
Comment (alphapapa) [inline sites/github.styl]: Without having investigated it myself, is this the best way to do this?  I'm guessing that I set `.octicon-check` to white for when it appears on a green background, IIRC.  Do we need to use a more specific selector for when it should be green?

OTOH maybe the CSS has changed and this is now the right way to do it.  :)

**Comment by alphapapa** [comment]
Hi Jay,

Thank you very much, this looks great!  I made a few comments about minor changes to make it a bit more idiomatic.  If you don't mind making those, I'll be glad to merge this right away!

File: sites/github.styl (hunk 3 above)
Comment (jgkamat) [reply sites/github.styl]: Ah, this is a mistake, I didn't know what the difference was, correcting!

Comment (alphapapa) [reply sites/github.styl]: No prob.  Indentation with selectors is equivalent to putting them on the same line separated by a space in regular CSS, so this selector only applies to `.branch-action-icon` classed items that are children of the `-state-*` selectors above it.

File: sites/github.styl (hunk 6 above)
Comment (jgkamat) [reply sites/github.styl]: This is a bit finiky. Github uses the `.text-green` selector to make the checks green. I tried overriding it, but it seemed the `.octicon` general selector was still overriding it. `.octicon-check` was the most specific selector I could find which worked. 

I don't *think* that the check is ever on a green background, I've only seen it on the background, like in branches. 

If there was a way of 'turning off' the `.octicon` custom style just for a class, then we could do it this way (but I don't know if that's possible).

**Comment by jgkamat** [comment]
Sorry about all the problems, this is my first time working with css, so I guess that's to be expected :P 

Let me know if you spot anything else wrong

>Did you try using white instead of black? I wonder if that might look more consistent with the overall theme. Let me know what you think.

I agree, white does look better, I'll switch it

>Also, on the yellow icon next to "Changes approved", could you make the icon white against the yellow background? 
Done! :)

File: sites/github.styl (hunk 6 above)
Comment (alphapapa) [reply sites/github.styl]: Ok, sounds good, thanks!

**Comment by alphapapa** [comment]
> Sorry about all the problems, this is my first time working with css, so I guess that's to be expected :P

Not at all!  You did a great job.  I'm very impressed for this being your first time working with CSS, I'd never have guessed, especially with your learning Stylus at the same time!  Stylus is powerful but so concise that it can be confusing and require some trial-and-error to figure out.

Thank you very much, and please feel free to submit more PRs in the future!
